docker-compose.yaml

```diff
@@ -14,7 +14,7 @@ services:
 
   postgres:
     container_name: postgres
-    image: postgres:14.5-alpine
```

Comment:
Do you think we should be allowing automatic major version updates to postgres here?

Reply:
I don't have a strong opinion, usually I just prefer running on latest and get ahead of any issues. If we prefer to pin an old version to make sure stuff "still works" there it's fine too